dts/bindings/misc/nordic,ironside-ipc.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: IPC configuration for Nordic IronSide SE
+
+compatible: "nordic,ironside-se-ipc"
```

Comment:
typo, I think, compatible `"nordic,ironside-se-ipc"` does not match `nordic_ironside_ipc`